frontend/src/components/ParticipantInfo.js

```diff
@@ -93,134 +85,18 @@ const ParticipantInfo = observer(() => {
-                Black or African American
-              </MenuItem>
-              <MenuItem value={"latino"}>Hispanic or Latino</MenuItem>
-              <MenuItem value={"native american"}>Native American</MenuItem>
-              <MenuItem value={"white (caucasian)"}>
-                White or Caucasian
-              </MenuItem>
-              <MenuItem value={"other"}>Other</MenuItem>
-            </Select>
-          </FormControl>
-        </Grid>
-        <Grid item xs={12} sm={6}>
-          <FormControl>
-            <InputLabel id="participant-gender">Select Gender</InputLabel>
-            <Select
-              required
-              value={existingParticipant.gender}
-              onChange={e => participantStore.setGender(e.target.value)}
-              labelId="participant-gender"
-            >
-              <MenuItem value={"male"}>Male</MenuItem>
-              <MenuItem value={"female"}>Female</MenuItem>
-              <MenuItem value={"mtf"}>Male to Female</MenuItem>
-              <MenuItem value={"ftm"}>Female to Male</MenuItem>
-              <MenuItem value={"gender queer"}>Gender Queer</MenuItem>
-              <MenuItem value={"other"}>Other</MenuItem>
-            </Select>
-          </FormControl>
-        </Grid>
-        <Grid item xs={12} sm={6}>
-          <FormControl component="fieldset">
-            <FormLabel component="legend">Has Insurance?</FormLabel>
-            <RadioGroup
-              aria-label="insurance"
-              name="hasInsurance"
-              value={existingParticipant.is_insured ? "yes" : "no"}
-              onChange={e =>
-                participantStore.setIsInsured(
-                  e.target.value === "yes" ? true : false
-                )
-              }
-              style={{ display: "inline" }}
-            >
-              <FormControlLabel value="yes" control={<Radio />} label="Yes" />
-              <FormControlLabel value="no" control={<Radio />} label="No" />
-            </RadioGroup>
-          </FormControl>
-        </Grid>
-        <Grid item xs={12} sm={6}>
-          <FormControl>
-            <InputLabel id="insurance-select">Select Insurance</InputLabel>
-            <Select
-              value={existingParticipant.insurer}
-              onChange={e => participantStore.setInsurer(e.target.value)}
-              labelId="insurance-select"
-            >
-              {insurers.map((company, index) => (
-                <MenuItem
-                  key={index}
-                  value={insurers && insurers.length > 0 ? company.id : ""}
-                >
-                  {insurers && insurers.length > 0 ? company.name : ""}
-                </MenuItem>
-              ))}
-            </Select>
-          </FormControl>
-        </Grid>
+        <ParticipantForm
+          insurers={insurers}
+          participantInfo={existingParticipant}
+          setRace={value => participantStore.setRace(value)}
```

Comment:
Id recommend at this point just letting `ParticipantForm` have access to the `particpantStore` unless we expect these props to be reusuable with different store types.

Reply:
ticket #321 describes a reusable handle change method, which would cut down on the number of function props. we wont be able to create methods for all the possible questions. Maybe that changes your opinion? this form doesn't need to be dynamically built but  most of the others will. 

TLDR this can stay stateful form but the dynamic ones probably shouldnt be

Reply:
I'm going to merge and revisit this later to keep the ball rolling